extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/SparkRangerAdminPluginSuite.scala

```diff
@@ -50,11 +50,14 @@ class SparkRangerAdminPluginSuite extends AnyFunSuite {
     }
     assert(getMaskingExpr(buildAccessRequest(bob, "value1")).get === "md5(cast(value1 as string))")
     assert(getMaskingExpr(buildAccessRequest(bob, "value2")).get ===
-      "regexp_replace(regexp_replace(regexp_replace(value2, '[A-Z]', 'X'), '[a-z]', 'x')," +
-      " '[0-9]', 'n')")
+      "regexp_replace(regexp_replace(regexp_replace(regexp_replace(value2, '[A-Z]', 'X')," +
+      " '[a-z]', 'x'), '[0-9]', 'n'), '[^A-Za-z0-9]', 'U')")
     assert(getMaskingExpr(buildAccessRequest(bob, "value3")).get contains "regexp_replace")
     assert(getMaskingExpr(buildAccessRequest(bob, "value4")).get === "date_trunc('YEAR', value4)")
-    assert(getMaskingExpr(buildAccessRequest(bob, "value5")).get contains "regexp_replace")
+    assert(getMaskingExpr(buildAccessRequest(bob, "value5")).get ===
+      "concat(regexp_replace(regexp_replace(regexp_replace(regexp_replace(" +
+      "left(value5, length(value5) - 4), '[A-Z]', 'X'), '[a-z]', 'x')," +
+      " '[0-9]', 'n'), '[^A-Za-z0-9]', 'U'), right(value5, 4))")
```

Comment:
I'm not sure about what's tested here. can we use plain value for assertion ?

Reply:
This is a test of the correctness of the getMaskingExpr method for the MASK_SHOW_LAST_4 rule, referring to the assert test MASK rule

Reply:
> can we use plain value for assertion

I'm sorry, I didn't understand the meaning of this sentence.

Reply:
I don't understand this test assertion.

Reply:
Originally, we used a simple 'contains' method to check. I think this can be improved by using '===' directly, which can more accurately determine whether the method of generating MASK_SHOW_LAST_4 based on this rule is correct or not.

Reply:
From my view, I would like to see a clear assertion with the pair of a concrete expected value  and a code to run. In current implementation , both sides on the assertion contain evaluation.
WDYT ? @yaooqinn

Reply:
I'm OK with the current implementation.

These tests are unimportant as we have e2e test cases. They are introduced when the data masking is not fully implemented. And at that time, I want to ensure the `getMaskingExpr` works well.

> both sides on the assertion contain evaluation.

I do not get your point. The author only uses '+' for newline.